node/src/container_chain_spawner.rs

```diff
@@ -172,6 +172,11 @@ impl ContainerChainSpawner {
             db_path.set_file_name(format!("full-container-{}", container_chain_para_id));
             container_chain_cli_config.database.set_path(&db_path);
 
+            // Delete existing database
+            if db_path.exists() {
+                std::fs::remove_dir_all(&db_path).expect("failed to remove old container chain db");
```

Comment:
This path is

```
Collator2002-01/data/containers/chains/simple_container_2002/db/full-container-2002
```

But we probably want to delete the parent folder:

```
Collator2002-01/data/containers/chains/simple_container_2002
```

But be careful to not delete more than we should, check where does this `simple_container_2002/db` come from